src/components/date-picker/date-picker.js

```diff
@@ -0,0 +1,227 @@
+    });
+    this._updateClassNames(calendar);
+    toDate.addEventListener('click', () => {
+      toDate.focus();
+      calendar.open();
+    });
+    datePickerIcon.addEventListener('click', () => {
+      calendar.open();
+    });
+    fromDate.addEventListener('focus', () => {
+      fromDate.value = fromDate.value.replace(/\s+/g, '');
+    });
+    toDate.addEventListener('focus', () => {
+      toDate.value = toDate.value.replace(/\s+/g, '');
+    });
+  }
+
+  _updateDatePickerPosition = (calendar) => {
+    const cal = calendar;
+    cal.calendarContainer.style.top = '67px';
+    cal.calendarContainer.style.left = '0px';
+  }
+
+  _updateClassNames = (calendar) => {
+    const calendarContainer = calendar.calendarContainer;
+    calendarContainer.classList.add(this.options.classCalendarContainer);
+    calendarContainer.querySelector('.flatpickr-month').classList.add(this.options.classMonth);
+    calendarContainer.querySelector('.flatpickr-weekdays').classList.add(this.options.classWeekdays);
+    calendarContainer.querySelector('.flatpickr-days').classList.add(this.options.classDays);
+    [...calendarContainer.querySelectorAll('.flatpickr-weekday')].forEach((item) => {
+      const currentItem = item;
+      currentItem.innerHTML = currentItem.innerHTML.replace(/\s+/g, '');
+      currentItem.classList.add(this.options.classWeekday);
+      if (currentItem.innerHTML === 'Thu' || currentItem.innerHTML === 'Th') {
+        currentItem.innerHTML = 'Th';
+      } else {
+        currentItem.innerHTML = item.innerHTML.charAt(0);
+      }
+    });
+    [...calendarContainer.querySelectorAll('.flatpickr-day')].forEach((item) => {
+      if ((item.classList.contains('selected') && item.classList.contains('inRange'))
+         || (item.classList.contains('selected') && item.classList.contains('endRange'))) {
+        if (item.previousElementSibling) {
+          item.previousElementSibling.classList.add(this.options.classPrevDay);
+        }
+      }
+      item.classList.add(this.options.classDay);
+    });
+  }
+
+  _updateInputFields = (selectedDates, isRange) => {
+    if (isRange) {
+      if (selectedDates.length === 2) {
+        this.element.querySelector(this.options.selectorDatePickerInputFrom).value = this._formatDate(selectedDates[0]);
+        this.element.querySelector(this.options.selectorDatePickerInputTo).value = this._formatDate(selectedDates[1]);
+      } else {
+        this.element.querySelector(this.options.selectorDatePickerInputFrom).value = this._formatDate(selectedDates[0]);
+      }
+    } else {
+      this.element.querySelector(this.options.selectorDatePickerInput).value = this._formatDate(selectedDates[0]);
+    }
+  }
+
+  _formatDate = (date) => {
+    const month = (date.getMonth() + 1) < 10
+    ? `0${(date.getMonth() + 1)}`
+    : (date.getMonth() + 1);
+    const day = date.getDate() < 10
+    ? `0${date.getDate()}`
+    : date.getDate();
+    return `${month} / ${day} / ${date.getFullYear()}`;
```

Comment:
Hi @asudoh , I changed the `formatDate` function a bit, do you think this is good enough or do we need to add more logic to this?